src/css/styles.css

```diff
@@ -0,0 +1,185 @@
+*,
+*:before,
+*:after {
```

Comment:
Hey Jesse, does this mean you re-added the css file to master?? We should have been able to move forward without a css file by doing any styling we needed to in the scss file. It doesn't look like this file is being accessed anywhere in the project, so should be good to leave out.